### DISCUSSION TITLE
Scripting: Percent sign in WebUI

### DISCUSSION THREAD

**Discussion by Ralph1234567** [arendst/Tasmota#21981] [Q&A] [answered]
Hi folks,

I like to have a percent sign after a value in the WebUI.

Entering a percent sign in the script editor lead to "???" in the WebUI as the editor awaits a variable, I guess.
Entering &#037; leads to a percent sign in the WebUI, so far so good.
But when entering the script Editor again, the &#037; is also converted here into a percent sign. So, after entering the wanted changes one have to reconvert all percent signs to &#037; again, before saving the script.

Any ideas to avoid that?

Many thanks
Ralph

**Comment by HofeBY** [comment, marked as answer]
%%

**Comment by Ralph1234567** [comment]
Oh god, I'm a bit ashamed ... is that simple ... :-)
Thanks a lot
